app/telegram/bot.py

```diff
@@ -9,3 +11,7 @@
 WEBHOOK_URL = settings.webhook_url
 
 telegram_bot = Bot(BOT_API, WEBHOOK_URL)
+
+loop = asyncio.get_event_loop()
```

Comment:
I think you can just `asyncio.run` the `set_webhook`

Reply:
It's impossible - asyncio.run() cannot be called from a running event loop (which started by FastAPI).